static/src/javascripts/projects/commercial/modules/article-body-adverts.ts

```diff
@@ -1,31 +1,37 @@
+import type { AdSize } from '@guardian/commercial-core';
+import { adSizes } from '@guardian/commercial-core';
+import { getBreakpoint, getViewport } from 'lib/detect-viewport';
 import config from '../../../lib/config';
 import fastdom from '../../../lib/fastdom-promise';
 import { mediator } from '../../../lib/mediator';
 import { spaceFiller } from '../../common/modules/article/space-filler';
-import { adSizes } from '@guardian/commercial-core';
-import { addSlot } from './dfp/add-slot';
-import { trackAdRender } from './dfp/track-ad-render';
-import { createAdSlot } from './dfp/create-slot';
-
 import { commercialFeatures } from '../../common/modules/commercial/commercial-features';
 import { initCarrot } from './carrot-traffic-driver';
-import { getBreakpoint, getTweakpoint, getViewport } from 'lib/detect-viewport';
+import { addSlot } from './dfp/add-slot';
+import { createAdSlot } from './dfp/create-slot';
+import { trackAdRender } from './dfp/track-ad-render';
 
-const isPaidContent = config.get('page.isPaidContent', false);
+const isPaidContent = config.get('page.isPaidContent', false) as boolean;
```

Comment:
I think we can pass the type as a generic...

```suggestion
const isPaidContent = config.get<boolean>('page.isPaidContent', false);
```